storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,66 @@
+import uuid from 'uuid/v4';
+import image from './nuxeo-logo.jpg';
+
+export default class Document {
+  // class methods
+  constructor(type = '', properties = {}, facets = [], permissions = [], isTrashed = false, isFavorite = false) {
+    this.type = type;
+    this.properties = properties;
+    this.facets = facets;
+    this.permissions = permissions;
+    this.isTrashed = isTrashed;
+    this.isFavorite = isFavorite;
+  }
+
+  trash() {
```

Comment:
Assuming that we'd have on the constructor an object for system properties, perhaps we should have a setter for the system properties as well? `.systemProperty('isTrashed', true)`